Easier fastboot testing setup

eriktrom opened the pull request:
Make it fast and easy to test node fastboot code

ember-cli is fast, nodemon re-runs the node fastboot tests when ember.debug.cjs.js is built

Includes docs on how to use

cc @tomdale

tomdale commented:
This is cool but `nodemon` seems like a heavy dependency just for auto-running tests, especially considering how big our dependency tree is already.

tomdale commented:
Also this should probably be `npm run test:node` for symmetry with the other tasks.

eriktrom committed:
easier node testing

- add npm tasks
	- npm run build:watch
	- npm run test:node
	- npm run test:node:watch

- add instructions to README on how to watch ember and run node tests
  on file change

eriktrom commented:
i updated the scripts to be more idiomatic - for some reason they fail the build for the last sauce labs suite - closing as this does add a lot more tasks to package.json - we can reopen if wanted
